### PR TITLE
Remove validator actions / change commit state logic

### PR DESCRIPTION
**Pull request by atvanguard**

### CODE REVIEW

File: contracts/BorValidatorSet.template [hunk 1]
```diff
@@ -20,12 +20,7 @@ contract BorValidatorSet is System, ValidatorSet {
   bytes32 public constant BOR_ID = keccak256("{{borChainId}}");
   uint8 public constant VOTE_TYPE = 2;
   uint256 public constant FIRST_END_BLOCK = {{firstEndBlock}};
-
-  // sprint
-  uint256 public sprint;
-
-  // pending span proposal
-  bool private _isSpanProposalPending;
+  uint256 public constant sprint = 64;
```
Comment (jdkanani) [inline contracts/BorValidatorSet.template:23]: Also use `sprint` in `currentSprint` method.

File: contracts/StateReceiver.sol [hunk 2]
```diff
@@ -1,64 +1,40 @@
...
-
-    // propose state by adding it into proposed states
-    IterableMapping.insert(proposedStates, stateId, true);
-  }
-
   // commit new state
   function commitState(
-    bytes calldata recordBytes
+    bytes calldata recordBytes,
+    uint256 syncTime
```
Comment (jdkanani) [inline contracts/StateReceiver.sol:20]: Suggestion: We should switch the position.

File: contracts/BorValidatorSet.template (hunk 1 above)
Comment (atvanguard) [reply contracts/BorValidatorSet.template]: Done.

File: contracts/StateReceiver.sol (hunk 2 above)
Comment (atvanguard) [reply contracts/StateReceiver.sol]: Done.

**Pull request merged**